e2e/wdio/core/tests/illustrated-message.e2e-spec.ts

```diff
@@ -0,0 +1,99 @@
+import { IllustratedMessagePo } from '../pages/illustrated-message.po';
+import {checkElementScreenshot, 
+    click, 
+    mouseHoverElement, 
+    saveElementScreenshot, 
+    isElementClickable, 
+    elementDisplayed,
+    doesItExist,
+    addIsActiveClass,
+    getElementArrayLength,
+    waitForElDisplayed} from '../../driver/wdio';
+
+describe('Illistrated-message tests', function() {
+    const illustratedMessagePage = new IllustratedMessagePo();
+    const {sceneAndSpotButtons, 
+        buttonDialog, 
+        dialogPopup,
+        closePopupSignButton,
+        closePopupButton} = illustratedMessagePage;
+
+    beforeAll(()=>{
+        illustratedMessagePage.open();                              
+    },1);
+
+    describe('verify buttons hover, focus and active state', function() {
+        it('should check back button hover state', () => {
+            const buttonsArrLength = getElementArrayLength(sceneAndSpotButtons);
+            for (let i = 0; buttonsArrLength > i; i++) {
+                checkButtonHoverState(sceneAndSpotButtons, 'HoverState'+ '-' + i, sceneAndSpotButtons, i);
+            }
+        });
+        it('should check back button focus state', () => {
+            const buttonsArrLength = getElementArrayLength(sceneAndSpotButtons);
+            for (let i = 0; buttonsArrLength > i; i++) {
+                checkButtonFocusState(sceneAndSpotButtons, 'FocusState'+ '-' + i, sceneAndSpotButtons, i);
+            }
+        });
+        it('should check back button active state', () => {
+            const buttonsArrLength = getElementArrayLength(sceneAndSpotButtons);
+            for (let i = 0; buttonsArrLength > i; i++) {
+                checkButtonActiveState(sceneAndSpotButtons, 'ActiveState'+ '-' + i, sceneAndSpotButtons, i);
+            }
+        });
+    });
+
+    it('should check RTL and LTR orientation', () => {              
+        illustratedMessagePage.checkRtlSwitch();
+    });
+
+    it('should check visual regression for all examples', () => {
+        illustratedMessagePage.saveExampleBaselineScreenshot();
+        expect(illustratedMessagePage.compareWithBaseline()).toBeLessThan(1);
+    });
+
+    it('should check is button clickable', () => {
+        const buttonsArrLength = getElementArrayLength(sceneAndSpotButtons);
+        for (let i = 0; buttonsArrLength > i; i++) {
+            expect(isElementClickable(sceneAndSpotButtons, i)).toBe(true, `Button ${i} is not clickable`);
+        }
+    });
+
+    it('should open dialog popup illustrated message', () => {
```

Comment:
would be great to have these in a describe like you did for the button checks
describe('check dialog example') ...